tests/codegen/asm/avr-clobbes.rs

```diff
@@ -0,0 +1,43 @@
+//@ assembly-output: emit-asm
```

Comment:
> avr-clobbes.rs

"r" appears to have been accidentally dropped from the filename. (avr-clobbers.rs should be correct)

Reply:
Well spotted, sorry! Fixed in the force-push